docs/_chapters/310-testing.md

```diff
@@ -147,7 +147,14 @@ As noted above, `biz.aQute.tester.junit-platform` requires JUnit Platform (and i
             url="file:///path/to/eclipse/";\
             location="${workspace}/cnf/cache/stable/EclipseLocal"
 ```
-Alternatively, it is not difficult to download the required (non-OSGi) modules from Maven Central and include them as-is on `-runpath`, or else (preferably) wrap them into bundles and include them in `-runrequires`/`-runbundles`.
+
+Alternatively, it is not difficult to download the required (non-OSGi) modules from Maven Central and include them as-is on `-runpath`, or else (preferably) wrap them into bundles and include them in `-runrequires`/`-runbundles`. 
+As of JUnit 5.6, the JUnit jars already have the OSGi metadata and so can be used as bundles 
+direct from Maven Central.
+
+Also note that unfortunately, due to a bug in `biz.aQute.tester.junit-platform`, Bndtools 5.0 
+*does not work with JUnit 5.5+*. This has already been fixed in the main development branch and we expect the 
+fix to be included in Bndtools 5.0.1 (to be released soon).
```

Comment:
5.0.1 is possible. I have made a test build. But we still need to collectively discuss and agree on a 5.0.1 release. So is it safer to simply say the release after 5.0.